index.html

```diff
@@ -29,13 +14,10 @@
       crossorigin="anonymous"
       referrerpolicy="no-referrer"
     />
-    <link rel="stylesheet" type="text/css" href="/ProjectUI_Navbar/style.css" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
```

Comment:
You were using absolute paths to access assets. I've changed it to relative paths. Avoid using absolute paths. Always prefer relative paths.